newIDE/app/src/ResourcesList/LocalResourceSources.js

```diff
@@ -207,7 +224,13 @@ const selectLocalResourcePath = (
 
     const properties = ['openFile'];
     if (options.multiSelections) properties.push('multiSelections');
-    const projectPath = path.dirname(project.getProjectFile());
+    let projectPath = path.dirname(project.getProjectFile());
+
+    // Load locally stored lastOpenedPath and update projectPath if not undefined
+    let lastOpenedPath = _loadLastOpenedPath();
```

Comment:
This is loading from the disk (localStorage) everytime we call this. We should avoid this.

Reply:
I have updated it.